app/views/shared/_sign_in_form.html.erb

```diff
@@ -0,0 +1,35 @@
+<%= form_for resource, as: 'user', url: user_session_path, html: {id: "sign_in_form"} do |f| %>
+  <% if Rails.application.config.shibboleth_enabled %>
+    <!-- Shibboleth DS enabled -->
+    <h3 class="center-text"><%= _('Sign in with') %></h3>
+
+    <div class="form-group">
+      <span class="center-block">
+        <% if request.fullpath != "/users/sign_up?nosplash=true" && session[:shibboleth_data].nil? then%>
+          <% target = (Rails.application.config.shibboleth_use_filtered_discovery_service ? shibboleth_ds_path : user_shibboleth_omniauth_authorize_path) %>
+          <a href="<%= target %>" class="btn btn-default"><%= _('Your Institution') %></a>
+        <%else%>
+          <%= f.hidden_field :shibboleth_id, :value => session[:shibboleth_data][:uid] %>
+        <%end%>
+      </span>
+    </div>
+
+    <h3 class="center-text">- <%= _('or') %> -</h3>
+  <% end %>
+  
+  <div class="form-group">
+    <%= f.label(:email, _('Email'), class: 'control-label') %>
+    <%= f.email_field(:email, class: 'form-control', "aria-required": true) %>
+    <%= f.check_box :remember_me, class: 'form-control' %>

```

Comment:
This checkbox does not appear inlined with the label. I think bootstrap got a default class for aligning input checkbox together with label, see the example here (https://getbootstrap.com/docs/3.3/css/#forms-example)